src/main-process/main.js

```diff
@@ -66,7 +66,7 @@ app.on('ready', () => {
     '48': nativeImage.createFromPath(path.join(logosPath, 'museeks-48.png')),
     '32': nativeImage.createFromPath(path.join(logosPath, 'museeks-32.png')),
     'ico': nativeImage.createFromPath(path.join(logosPath, 'museeks.ico')),
-    'tray': nativeImage.createFromPath(trayIconPath).resize({ width: 24, height: 24 }),
+    'tray': nativeImage.createFromPath(trayIconPath),

```

Comment:
So I remember adding this line when I was on Elementary because otherwise the. icon would actually take 256px in the top bar.

I will try to reproduce it.

Reply:
looks like Skype (the latest version is based on nodejs) has the same issue, also MailSpring, Harmony just need to add some borders cuz it's icon is too big, but Slack works great.